#goal         
bool calib_request
---
#result
float32[] left_hand_pose
float32[] right_hand_pose
---
#feedback
bool calib_success
